draft-ietf-cbor-7049bis.md

```diff
@@ -2094,32 +2094,36 @@ Author/Change Controller:
 
 # Security Considerations {#securitycons}
 
-A network-facing application can exhibit vulnerabilities in its
-processing logic for incoming data. Complex parsers are well known as
-a likely source of such vulnerabilities, such as the ability to
-remotely crash a node, or even remotely execute arbitrary code on it.
-CBOR attempts to narrow the opportunities for introducing such
-vulnerabilities by reducing parser complexity, by giving the entire
-range of encodable values a meaning where possible.
-
-Resource exhaustion attacks might attempt to lure a decoder into
-allocating very big data items (strings, arrays, maps) or exhaust the
-stack depth by setting up deeply nested items.  Decoders need to have
-appropriate resource management to mitigate these attacks.  (Items for
-which very large sizes are given can also attempt to exploit integer
-overflow vulnerabilities.)
-
-Protocols that are used in a security
-context should be defined in such a way that potential multiple
-interpretations are reliably reduced to a single one.
-For example, an attacker could make use of duplicate keys in
-maps or precision issues in numbers to make one decoder base its
-decisions on a different interpretation than the one that will be used
-by a second decoder.
-To facilitate
-this, encoder and decoder implementations used in such contexts should
-provide at least one strict mode of operation ({{strict-mode}}).
-
```

Comment:
From a security point of view.  This paragraph is something that is of importance.  It is really necessary to make sure that duplicate keys can be prevented.  Allowing duplicate tags is not necessarily verboten.